docs/build/build-binary-go.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Build Binary - Go 
+id: build-binary-go
+---
+
+This guide explains how to install the cudos-noded binary with Go. 
+
+These instructions are written based on a Google Cloud VM instance running on Ubuntu 20.04 LTS
+x86/64, amd64 focal image built on 2022-09-05, supports Shielded VM features.
+
+## Supported OS
+
+### Go
+
+* Ubuntu 20.04 LTS 
+
+## 00 Install prerequisites
+
+### Install `build-essential`
+
+```shell
+root@instance-1:~# sudo apt update
+root@instance-1:~# sudo apt install build-essential
+```
+
+### Install `Go`
+
+:::warning Remove any previous Go installation
+Delete the `/usr/local/go` folder (if it exists)
+
+```shell
+root@instance-1:~# sudo rm -rf /usr/local/go
+```
+:::
+
+1. Download and extract Go in `/usr/local/go`
+
+```shell
+root@instance-1:~# wget https://go.dev/dl/go1.19.1.linux-amd64.tar.gz
+root@instance-1:~# sudo tar -C /usr/local -xvf go1.19.1.linux-amd64.tar.gz
+````
+
+2. Add `/usr/local/go/bin` to the `PATH` environment variable.
+
+```shell
+echo "PATH=$PATH:/usr/local/go/bin:$HOME/go/bin" >> ~/.profile
```

Comment:
Don't do this. The path will be expanded at the time this command is executed, not when the profile is executed. That will mean the user's path will never pick up any changes from global OS configuration.

Additionally, because it isn't quoted correctly, if the PATH or HOME contain spaces, or any other characters the shell will interpret specially in this context, this may simply break the user's profile.